`container` does not build as a standalone service

Compiling with `vcpkg`.  Could be a packaging problem, but I am making a note here.

```
FAILED: google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.grpc.pb.cc.o 
/usr/bin/c++ -DCARES_STATICLIB -isystem /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container -isystem /usr/local/google/home/coryan/vcpkg-develop/installed/x64-linux/include -isystem /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/external/googleapis -fPIC -O3 -DNDEBUG -std=gnu++11 -MD -MT google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.grpc.pb.cc.o -MF google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.grpc.pb.cc.o.d -o google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.grpc.pb.cc.o -c /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container/google/container/v1/cluster_service.grpc.pb.cc
In file included from /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container/google/container/v1/cluster_service.grpc.pb.cc:5:
/usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container/google/container/v1/cluster_service.pb.h:46:10: fatal error: google/rpc/status.pb.h: No such file or directory
   46 | #include "google/rpc/status.pb.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[272/796] /usr/bin/c++ -DCARES_STATICLIB -isystem /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container -isystem /usr/local/google/home/coryan/vcpkg-develop/installed/x64-linux/include -isystem /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/external/googleapis -fPIC -O3 -DNDEBUG -std=gnu++11 -MD -MT google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.pb.cc.o -MF google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.pb.cc.o.d -o google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.pb.cc.o -c /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container/google/container/v1/cluster_service.pb.cc
FAILED: google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.pb.cc.o 
/usr/bin/c++ -DCARES_STATICLIB -isystem /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container -isystem /usr/local/google/home/coryan/vcpkg-develop/installed/x64-linux/include -isystem /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/external/googleapis -fPIC -O3 -DNDEBUG -std=gnu++11 -MD -MT google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.pb.cc.o -MF google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.pb.cc.o.d -o google/cloud/container/CMakeFiles/google_cloud_cpp_container_protos.dir/google/container/v1/cluster_service.pb.cc.o -c /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container/google/container/v1/cluster_service.pb.cc
In file included from /usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container/google/container/v1/cluster_service.pb.cc:4:
/usr/local/google/home/coryan/vcpkg-develop/buildtrees/google-cloud-cpp/x64-linux-rel/google/cloud/container/google/container/v1/cluster_service.pb.h:46:10: fatal error: google/rpc/status.pb.h: No such file or directory
   46 | #include "google/rpc/status.pb.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

--- a/google/cloud/container/CMakeLists.txt
+++ b/google/cloud/container/CMakeLists.txt
@@ -50,7 +50,8 @@ target_link_libraries(
            google-cloud-cpp::api_client_protos
            google-cloud-cpp::api_field_behavior_protos
            google-cloud-cpp::api_http_protos
-           google-cloud-cpp::rpc_code_protos)
+           google-cloud-cpp::rpc_code_protos
+           google-cloud-cpp::rpc_status_protos)
 
 file(
     GLOB source_files

